_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,496 @@
+---
+id: efbfc7
+name: Auto-updating text content can be paused or stopped
+rule_type: atomic
+description: |
+  This rule checks that there are mechanisms to pause or stop the auto-updating of text content
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+acknowledgements:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [visible][] [text node][] in the [flat tree][] of an [HTML document][] if:
+
+- the content of the [text node][] changes through an action that was not initiated by the user;
```

Comment:
I don't think changing text nodes is a reliable way to determine there's a problem. For one, text can be added to the text node. That's a change that doesn't cause a problem. Or perhaps the current text is put into another text node, while the current text node is updated. This is pretty common stuff with virtual DOMs.

Another thing this doesn't consider is that the text node could be removed entirely. I wouldn't consider that a change, but it can be a failure.

Reply:
Given that the SC says "Auto-updating refers to content that updates or disappears" I'm updating the applicability to "changes, is replaced or removed"